src/bindings/common.ts

```diff
@@ -68,3 +68,9 @@ export function SHA256(text: string | number | undefined) {
   const result = Buffer.from(digest).toString('hex');
   return result;
 }
+
+export function containsAll(arr1: any[], arr2: any[]): boolean {
```

Comment:
here we are saying arr1 is subset of arr2 right? add that as comment

Reply:
ok